app/src/main/java/com/jerboa/ui/components/drawer/Drawer.kt

```diff
@@ -70,6 +71,7 @@ import com.jerboa.ui.theme.muted
 @Composable
 fun Drawer(
     myUserInfo: MyUserInfo?,
+    follows: List<CommunityFollowerView>,
```

Comment:
Use immutableList if you pass list through composables, list is deemed unstable by compose see https://issuetracker.google.com/issues/199496149